gfx/tileset_palette_maps.inc

```diff
@@ -0,0 +1,115 @@
+tilepal: MACRO
+; used in gfx/tilesets/*_palette_map.asm
```

Comment:
They're gfx/tilesets/\*_palette_map.inc now.

Reply:
This is fixed already